diffus/Cargo.toml

```diff
@@ -50,6 +50,7 @@ uuid-impl = [ "uuid" ]
 snake_case-impl = [ "snake_case" ]
 
 serialize-impl = [
+    "diffus-derive/serialize-impl",
```

Comment:
I thought about adding this but refrained from doing so as it means that `serialize-impl` implies the `derive` feature which doesn't seem entirely right.

Reply:
As discussed offline, this seems like a corner-case issue, so we're going with it!